controller/lib/hal/hal.h

```diff
@@ -110,15 +110,25 @@ enum class AnalogPin {
   OUTFLOW_PRESSURE_DIFF,
 };
 
-// IDs of the digital pins that can be used for pulse-width modulation.
-// https://github.com/arduino/ArduinoCore-avr/blob/257ee3f/variants/standard/pins_arduino.h#L35
-enum class PwmPinId {
-  PWM_3 = 3,
-  PWM_5 = 5,
-  PWM_6 = 6,
-  PWM_9 = 9,
-  PWM_10 = 10,
-  PWM_11 = 11,
+// Pulse-width modulated outputs from the controller.  These can be set to
+// values in [0-255].
+//
+// Pins default to INPUT, so be sure to update HalApi::init() if you add a new
+// pin here!
+enum class PwmPin {
+  // Controls the fan speed.
+  BLOWER,
+};
+
+// Binary outputs from the controller -- these are booleans, HIGH or LOW.
```

Comment:
s/Binary outputs/Binary pins set by the controller, to make it clear that this isn't just something the controller computes (e.g. flow) but something the controller sets.